x-pack/platform/plugins/shared/alerting/server/routes/rule/apis/create/create_rule_route.ts

```diff
@@ -83,6 +84,21 @@ export const createRuleRoute = ({ router, licenseState, usageCounter }: RouteOpt
           });
 
           try {
+            const ruleType = ruleTypes.get(createRuleData.rule_type_id);
+
+            /**
+             * Throws a bad request (400) if the rule type is internallyManaged
```

Comment:
why 400? I think 403 would be more fitting:
 
In https://www.rfc-editor.org/rfc/rfc9110#status.403
```
However, a request might be forbidden for reasons unrelated to the credentials.
```